### ISSUE TITLE
[BUG] Since the update from 1.14.4 to 1.14.5 there are no more entries in the paperless.log

### ISSUE THREAD

**Issue by wastlkl** [paperless-ngx/paperless-ngx#3460]
### Description

<img width="1258" alt="Bildschirmfoto 2023-05-25 um 11 52 40" src="https://github.com/paperless-ngx/paperless-ngx/assets/32601347/4a2a4f24-77ef-4a26-b997-20b9e237a160">

### Steps to reproduce

The problem is reproducible.
Restoring the VM and updating again brought the same result

### Webserver logs

```bash
no
```

### Paperless-ngx version

1.14.5

### Host OS

Debian GNU/Linux 11 (bullseye)

### Installation method

Docker - official image

### Browser

Safari 16.5

**Comment by stumpylog** [comment]
That's an odd one.  Nothing changed in logging I can recall. And my own production instance is logging fine.  

Maybe check permissions?  Are you looking at the actual file or via the frontend?

**Comment by shamoon** [comment]
Yea that picture is the UI, would be curious if it differs from the file.

Have you tried removing the file and re-creating the container?

And just to check, things that get added to the log are e.g. document consumption, classifier training, and some startup messages

**Comment by wastlkl** [comment]
Thank You for Your answers.
Currently I have reset the vm and thus back to version 1.14.4. Then I updated debian. After that everything still worked. When I finally did the pull, the logging no longer worked. Changes are no longer written either in the protocol panel or in the file.
I've got one more question: How can the update change the permissions, I haven't made any changes?

**Comment by stumpylog** [comment]
An update doesn't change permissions.  But other things on a system always can.  Check if the paperless user can write the the log directory, then the log file itself, etc.

**Comment by shamoon** [comment]
And you mentioned re-setting the vm but I would also try to completely remove the docker container and re-create it

**Comment by wastlkl** [comment]
Thank you very much again for yours answers.
I deleted and recreated the docker container but my data drive remained. However, this did not help.
For me i'm using a workaround now. Since it may be a question of access rights, I have now connected the data via nfs instead of smb. Now logging works even after updating to v.1.14.5.
Nevertheless, I will take a look at the topic with smb again when the time comes.
